agendas/2022/2022-01-06.md

```diff
@@ -96,7 +96,9 @@ This is an open meeting in which anyone in the GraphQL community may attend.
 | Lee Byron          | @leebyron       | GraphQL Foundation | San Francisco, CA, US
 | Yaacov Rydzinski   | @yaacovCR       | Individual         | Neve Daniel, IL
 | Stephen Spalding   | @fotoetienne    | Netflix            | Los Gatos, CA, US
+| Ilya Solovyiov     | @ilslv          | [`juniper`]        | Minsk, Belarus, BY
 
+[`juniper`]: https://github.com/graphql-rust/juniper
```

Comment:
We don't normally add links here, so for the sake of expedience (so I can merge the PR ASAP) I've removed the link; but I'd like to know from @leebyron whether linking here should be allowed/encouraged - I see arguments for both sides, but am erring towards not allowing them to keep agenda's clean.

```suggestion
| Ilya Solovyiov     | @ilslv          | juniper            | Minsk, Belarus, BY
```